pkg/domain/events.go

```diff
@@ -63,9 +63,14 @@ func (s StateDiffDeterminedEvent) Name() string {
 }
 
 func (s StateDiffDeterminedEvent) Message() string {
-	toInstall, toUpgrade, toUninstall, others := s.StateDiff.DoguDiffs.Statistics()
-	return fmt.Sprintf("state diff determined: %d dogu diffs (%d to install, %d to upgrade, %d to delete, %d others)",
-		len(s.StateDiff.DoguDiffs), toInstall, toUpgrade, toUninstall, others)
+	doguInstall, doguUpgrade, doguUninstall, others := s.StateDiff.DoguDiffs.Statistics()
+	// TODO how should the component state diff be incorporated into this event output? An own event for each of them, or one for all ⚔️?
```

Comment:
Newlines don't bode well in events. Thus my `TODO` to give a heads-up.

My educated guess is: they will be shown but I haven't read on how they will be rendered or if there are any best practices. 

BTW: This will get worse when we enter registry diffing into the same event

Reply:
I am also not very happy with the structure that different diffs (dogu/comps/registry) might be calculated and then all pressed into the same event. I feel this should be a different event for each of them...

Reply:
- [x] done